pkgs/development/python-modules/auditwheel/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, patchelf
+, pbr
+, pretend
+, pyelftools
+, pytestCheckHook
+, pythonAtLeast
+}:
+
+buildPythonPackage rec {
+  pname = "auditwheel";
+  version = "4.0.0";
+
+  disabled = ! pythonAtLeast "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "03a079fe273f42336acdb5953ff5ce7578f93ca6a832b16c835fe337a1e2bd4a";
+  };
+
+  buildInputs = [
+    pbr
+  ];
+
+  propagatedBuildInputs = [
+    pyelftools
+  ];
+
+  # integration tests require docker and networking
+  disabledTestPaths = [ "tests/integration" ];
+
+  checkInputs = [
+    pretend
+    pytestCheckHook
+  ];
+
+  makeWrapperArgs = [
+    ''--prefix PATH : ${patchelf}/bin''
```

Comment:
This does not work for Python modules when they are imported. If this isn't meant to be imported, move the expression to `pkgs/tools/...` and use `buildPythonApplication`. Otherwise, you have to patch the path to the binary into the source.

Reply:
Thanks for the heads up. I now changed it to buildPythonApplication and moved it to tools/package-management/